src/components/frame/Extensions/layouts/ColonyLayout.tsx

```diff
@@ -80,14 +81,24 @@ const ColonyLayout: FC<PropsWithChildren> = ({ children }) => {
     return <Navigate to={NOT_FOUND_ROUTE} />;
   }
 
-  const txButtons = isMobile
-    ? [
-        groupState === TransactionGroupStates.SomePending && <PendingButton />,
-        groupState === TransactionGroupStates.AllCompleted && (
-          <CompletedButton />
-        ),
-      ]
-    : null;
+  const txButtons = [
+    groupState === TransactionGroupStates.SomePending && (
+      <PendingButton
+        text={isMobile ? undefined : { id: 'button.pending' }}
+        className={clsx({
+          '!min-w-0': isMobile,
+        })}
+      />
+    ),
+    groupState === TransactionGroupStates.AllCompleted && (
+      <CompletedButton
+        text={isMobile ? undefined : { id: 'button.completed' }}
+        className={clsx({
+          '!min-w-0': isMobile,
+        })}
+      />
+    ),
+  ];
```

Comment:
Do these need to be two separate components? Can it not just be one component with the pending and completed states?